backend/src/main.py

```diff
@@ -23,11 +24,25 @@
 import json
 from sqlalchemy.ext.serializer import loads, dumps
 
+import entities.mail
+
 app = Flask(__name__)
+
+app.config['MAIL_SERVER'] = 'smtp.gmail.com'
+app.config['MAIL_PORT'] = 465
+app.config['MAIL_USERNAME'] = 'indium@gmail.com'
+app.config['MAIL_PASSWORD'] = 'xxxxxxxxx'
```

Comment:
shouldnt this be an actual value instead of xxxxx.

Reply:
we don't have default credential. we can't share our credentials.